open-metadata-resources/open-metadata-labs/egeria-server-config.ipynb

```diff
@@ -447,6 +447,7 @@
     "configureAccessService(adminPlatformURL, adminUserId, mdrServerName, 'asset-consumer', accessServiceOptions)\n",
     "\n",
     "accessServiceOptions[\"DefaultZones\"] = [ \"quarantine\" ]\n",
+    "accessServiceOptions[\"PublishZones\"] = [ \"data-lake\" ]\n",
```

Comment:
I notice that the [docs](https://egeria-project.org/education/egeria-dojo/developer/egeria-platform-report-mds1-configured/?h=publishzones) say: `SupportedZones: [quarantine, trash-can, data-lake]
PublishZones: [data-lake, personal-files, personal-files]
DefaultZones: [quarantine]`. This mismatches what we have here for cocoMDS1.

I suggest raising an issue associated with this pr, with some description.